src/main/java/org/opentripplanner/routing/flex/GtfsFlexGraphModifier.java

```diff
@@ -151,8 +129,6 @@ public void createForwardHops(RoutingRequest request) {
         RoutingRequest forward = request.clone();
         forward.setMode(getMode());
         forward.setArriveBy(false);
-        // allow discovery of stations even with car mode
-        forward.enterStationsWithCar = true;
```

Comment:
Do you know why it was set to true?

Reply:
It was somehow related to `call-and-ride`.